utils/node-fuzzer/fuzz/fuzz_targets/main.rs

```diff
@@ -0,0 +1,44 @@
+#![no_main]
```

Comment:
Probably we need license header there and in some other fiels.

Reply:
done